sdk/src/main/java/com/mapbox/maps/renderer/MapboxSurfaceRenderer.kt

```diff
@@ -7,16 +7,27 @@ internal open class MapboxSurfaceRenderer : MapboxRenderer {
 
   private var createSurface = false
 
+  override val widgetRenderer: MapboxWidgetRenderer
+
   constructor(antialiasingSampleCount: Int) {
+    widgetRenderer = MapboxWidgetRenderer(
+      translucentSurface = false,
+      antialiasingSampleCount = antialiasingSampleCount,
+    )
```

Comment:
here and in other places - I still see no sense in creating it __not__ lazily. 99% of use-cases do not need those renders. can we revisit it somehow and not even create instance if it's not needed?